scp/lib/ui/cards.dart

```diff
@@ -9,9 +9,18 @@ import 'package:scp/firebase/firebaseDBHandler.dart';
 import 'package:scp/utils/routes.dart';
 import 'package:scp/utils/sizeConfig.dart';
 import 'package:shared_preferences/shared_preferences.dart';
+import 'package:url_launcher/url_launcher.dart';
 import '../utils/models.dart';
 import 'package:scp/upload_image.dart';
 
+_launchLink(String url) async {
```

Comment:
Move the function after main class

Reply:
@aryandas12 
This is a redundant function. The same function has been made at several places in the project and is being used. If you're still working on this, please try to keep this one function in a single separate file and reuse it at every other place. Comment here if you face any problem.